cmake/cpack_settings.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# CPack variables for the regular OE SDK.
+include(InstallRequiredSystemLibraries)
+set(CPACK_PACKAGE_NAME "open-enclave")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Open Enclave SDK")
+set(CPACK_PACKAGE_CONTACT "openenclave@microsoft.com")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
+set(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE")
+set(CPACK_PACKAGE_VERSION ${OE_VERSION})
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libsgx-enclave-common (>=2.3.100.46354-1), libsgx-enclave-common-dev (>=2.3.100.0-1), libsgx-dcap-ql (>=1.0.100.46460-1.0), libsgx-dcap-ql-dev (>=1.0.100.46460-1.0), pkg-config")
```

Comment:
Just curious, why do these _depend_ on pkg-config instead of just recommending it? Do they need it at runtime to find other packages?

Reply:
Good point. We shouldn't need to depend on pkgconfig (i.e. it's not necessary to use the SDK).

Reply:
I created https://github.com/openenclave/openenclave/issues/2107 for it